cypress/support/config/settings.js

```diff
@@ -2586,11 +2586,11 @@ module.exports = () => ({
       storyPage: {
         environments: {
           live: {
-            paths: [],
+            paths: ['/kyrgyz/world-52917194'],
             enabled: false,
           },
           test: {
-            paths: [],
+            paths: ['/kyrgyz/world-23090100'],
             enabled: false,
           },
           local: {
```

Comment:
Can we add a local url too?

Reply:
Added [here](https://github.com/bbc/simorgh/pull/6788/commits/4767d5abefd627065a1a1f550058ea7a25951316).

Reply:
I think we need to regenerate the release info after adding the local URLs 👍